src/schemas/space.json

```
@@ -33,13 +33,13 @@
         "twitter": {
           "type": "string",
           "title": "twitter",
-          "pattern": "^[A-Za-z0-9_]*$",
+          "pattern": "^[A-Za-z0-9_-]*$",
```

Twitter dont allow dash
```suggestion
          "pattern": "^[A-Za-z0-9_]*$",
```